packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -129,6 +113,9 @@ export type KarmaChangeUpdateFrequency = UnionOf<typeof karmaChangeUpdateFrequen
 
 export interface KarmaChangeSettingsType {
   updateFrequency: KarmaChangeUpdateFrequency
+  /**
+   * Time of day at which daily/weekly batched updates are released. A number of hours [0,24), always in GMT.
```

Comment:
```suggestion
   * Time of day at which daily/weekly batched updates are released. A number of hours [0,24), always in UTC.
```

Reply:
Given that the schema field names are called e.g. `timeOfDayGMT` I think I'll keep this as GMT to avoid confusion